Unscheduled Care/2. Unscheduled Care outputs.R

```diff
@@ -885,16 +1571,23 @@ ppa <- arrow::read_parquet(paste0(import_folder, "ppa_smr.parquet")) %>%
 # % PPAs in locality under and over 65
 ppa_total <- ppa %>%
   rename(n = admissions) %>%
+<<<<<<< HEAD
   aggregate_usc_area_data()
 
+=======
+  aggregate_usc_area_data() %>%
+  drop_na(year)
+>>>>>>> de46a210b6bacab9b561a73d14eb2bb4891c79ba
```

Comment:
@JosieSteele - Some conflict markers were introduced (it's probably because I'd made some commits which you hadn't pulled... sorry!)